vyper/builtins/functions.py

```diff
@@ -1679,6 +1680,48 @@ def build_IR(self, expr, args, kwargs, context):
         )
 
 
+class RawCreate(_CreateBase):
+    _id = "raw_create"
+    _inputs = [("bytecode", BytesT(EIP_3860_LIMIT))]
+    _has_varargs = True
+
+    def _add_gas_estimate(self, args, should_use_create2):
+        return _create_addl_gas_estimate(EIP_170_LIMIT, should_use_create2)
+
+    def _build_create_IR(self, expr, args, context, value, salt, revert_on_failure):
+        initcode = args[0]
+        ctor_args = args[1:]
+
+        ctor_args = [ensure_in_memory(arg, context) for arg in ctor_args]
+
+        # encode the varargs
+        to_encode = ir_tuple_from_args(ctor_args)
+        type_size_bound = to_encode.typ.abi_type.size_bound()
+        bufsz = initcode.typ.maxlen + type_size_bound
+
+        buf = context.new_internal_variable(get_type_for_exact_size(bufsz))
+
+        ret = ["seq"]
+
+        with scope_multi((initcode, value, salt), ("initcode", "value", "salt")) as (
+            b1,
+            (initcode, value, salt),
+        ):
+            bytecode_len = get_bytearray_length(initcode)
+
+            maxlen = initcode.typ.maxlen
+            ret.append(copy_bytes(buf, bytes_data_ptr(initcode), bytecode_len, maxlen))
```

Comment:
it shouldn't be problematic that `copy_bytes` might pad to `ceil32`, right?

it might copy some dirty data, but if arguments are provided, those should overwrite it

if no args are provided, we still operate with `bytecode_len` which will lead to ignoring the dirty data